execution/gethexec/sequencer.go

```diff
@@ -972,7 +926,15 @@ func (s *Sequencer) createBlock(ctx context.Context) (returnValue bool) {
 	}
 
 	start := time.Now()
-	block, err := s.execEngine.SequenceTransactions(header, txes, hooks)
+	var (
+		block *types.Block
+		err   error
+	)
+	if s.config().EnableProfiling {
```

Comment:
nit: we could use a `config` copy that we already got in this scope to avoid calling the fetcher again
https://github.com/OffchainLabs/nitro/blob/874bea0365f6fcc581b4f018cb7bd7cea05b5caa/execution/gethexec/sequencer.go#L802